packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+The client-based rate-limiting framework can exist in one of three modes:
+
+| Mode                                     | Description                                                                                                          |
+| ---------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| **Enforce and log per client (recommended)** | Rate limiting is based on client-based rate-limiting values. Client-specific rate limiting violation information is logged by firing System Log events. |
+| **Log per client**                          | Rate limiting is still based on the [org-wide Rate Limit](/docs/reference/rate-limits/) values, but client-specific rate limiting violation information is logged by firing System Log events. |
+| **No action**                                | Rate limits aren't enforced at the client-specific level. Rate limiting is based on the [org-wide Rate Limit](/docs/reference/rate-limits/) values. |
+
+### System Log events
+
+When an individual user violates the per minute requests limit assigned to them, depending on which mode is set, the following System Log event is fired:
+
+* `system.client.rate_limit.violation`: This event is fired when the framework is in **Enforce and log per client** mode. This event is fired when a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.concurrency_rate_limit.violation`: This event is fired when the framework is in **Enforce and log per client** mode. This event is fired when a specific client, IP address, or device identifier combination makes more than two concurrent requests. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.rate_limit.notification `: This event is fired when the framework is in **Log per client** mode. This event is fired when a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.concurrency_rate_limit.notification`: This event is fired when the framework is turned on in **Log per client** mode. This event is fired when a specific client, IP address, Device token combination makes more than two concurrent requests. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, Device identifier, and the actual user if the user already has a valid session.
```

Comment:
Same change suggestions as from line 77 (shorten and do they see a notification).